components/fieldgroups/factor/factor-standard-fields.schema.json

```diff
@@ -0,0 +1,134 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/factor-standard-fields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Factor Standard Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/summarymetrics"],
+  "description": "A field group that defines standard factor fields including factor name, factor value, factor type, factor value type, and Granularity.",
+  "definitions": {
+    "factor-standard-fields": {
+      "type": "object",
+      "properties": {
+        "xdm:factorName": {
+          "title": "Factor Name",
+          "type": "string",
+          "description": "The name of the factor",
+          "examples": [
+            "CPI",
+            "Humidity",
+            "Sales Volume",
+            "Customer Satisfaction Score"
+          ],
+          "meta:titleId": "factor-standard-fields##xdm:factorName##title##98732",
+          "meta:descriptionId": "factor-standard-fields##xdm:factorName##description##99700"
+        },
+        "xdm:factorValue": {
+          "title": "Factor Value",
+          "type": "number",
+          "meta:xdmType": "double",
```

Comment:
`"meta:xdmType": "double"` is not supported; just `"type"\:"number"` is sufficient